src/components/sbb-link/sbb-link.tsx

```diff
@@ -46,14 +46,20 @@ export class SbbLink {
    */
   @Prop() public iconFlip?: boolean;
 
+  /**
+   * If this is set to true an span element will be used
+   * instead of a anchor or a button
+   */
+  @Prop() public staticSpan?: boolean = false;
```

Comment:
I'd prefer to just use `static` or was there a problem with that?

Reply:
Its a reserved keyword.